int_experian/int_experian_sgjc/cartridge/static/default/js/edqUtils.js

```diff
@@ -19,7 +19,8 @@ var vDefaultCountry,
 	edqDataSetUsage,
 	edqDataSetCode,
 	edqProWebCallbackValidation,
-	edqCustomCallbackName;
+	edqCustomCallbackName,
+	pageRestrictValidation = true;
```

Comment:
A comment on what does `pagerestrictValidation` does would be helpful.

Reply:
The issue was that when Email and Phone validation were enabled and their page restriction for unmatched results were enabled; if you set a non valid email but a valid phone the page restriction would take the result as true since it'll check individually the restriction and the phone validation is the second in execution; with this variable for this case we won't need the large "if" that I did before to check if the button was enabled we'll be checking if the variable is true or false.